src/Randomizer.Data/Configuration/Yaml/Sassy/responses.yml

```diff
@@ -462,20 +482,24 @@ Moods:
   - Text: I'm not talking with you right now
   - Text: Oh Im sorry? Do you need more from me than the everything I am already taking care of for you?
   - Text: I hear what you say about me behind my back
+  - Text: I saw that you've been playing other games recently. Do you know how that makes me feel?
 Hints:
   EnabledHints:
   - Text: Toggled hints on.
   - Text: Hints are now enabled.
   - Text: Hints are now on.
   - Text: I'll give you hints when asked.
   - Text: I am here to help
+  - Text: Why am I not surprised that you need assistance?
+    Weight: 0.2
   DisabledHints:
   - Text: Toggled hints off.
   - Text: Hints are no longer enabled.
   - Text: Hints are no longer on.
   - Text: I'll stop giving you hints.
   - Text: I can no longer assist
   - Text: Fine. Figure it out yourself.
+  - Text: Good. One less than I have to do.
```

Comment:
Should this be "one less thing"?

Reply:
Maybe one day I'll proofread text I write...